### DISCUSSION TITLE
API to create apps

### DISCUSSION THREAD

**Discussion by alexandrosk** [coollabsio/coolify#1353] [Feature Requests]
I would love to work on an api to create new instances and services easily so you don't have to use the UI and can setup automatic Webhooks etc

**Comment by agmyomyat** [comment]
this would be so cool. this is the only reason i am not yet using coolify. i want this so bad

**Reply by compgeniuses** [reply]
the new updates have added some PI functions.

**Reply by Wulfheart** [reply]
What doues PI mean?

**Reply by compgeniuses** [reply]
API

**Comment by kishaningithub** [comment]
A terraform provider for coolify will be pretty "cool" :-)
